CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 0.12.29+1
+## 0.12.30
```

Comment:
Why not 0.12.29+1?

Reply:
The pubspec was already at 0.12.30-dev.

Reply:
That's just because `0.12.29+1-dev` isn't valid. There aren't any changes in the changelog that would require a minor version bump.

Reply:
Ah, I see. Done.